src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-node-statistics.ts

```diff
@@ -0,0 +1,400 @@
+                    </div>`
+                  : ``}
+                ${wrValue.route_failed_between
+                  ? html`<div class="row">
+                      <span>
+                        ${this.hass.localize(
+                          "ui.panel.config.zwave_js.route_statistics.route_failed_between.label"
+                        )}<ha-help-tooltip
+                          .label=${this.hass.localize(
+                            "ui.panel.config.zwave_js.route_statistics.route_failed_between.tooltip"
+                          )}
+                        >
+                        </ha-help-tooltip
+                      ></span>
+                      <span
+                        >${this._computeDeviceNameById(
+                          wrValue.route_failed_between[0]
+                        )}<ha-svg-icon .path=${mdiSwapHorizontal}></ha-svg-icon
+                        >${this._computeDeviceNameById(
+                          wrValue.route_failed_between[1]
+                        )}</span
+                      >
+                    </div>`
+                  : ``}
+                ${wrValue.repeaters.length > 0
+                  ? html`<div class="row">
+                      <span>
+                        ${this.hass.localize(
+                          "ui.panel.config.zwave_js.route_statistics.repeaters.label"
+                        )}<ha-help-tooltip
+                          .label=${this.hass.localize(
+                            "ui.panel.config.zwave_js.route_statistics.repeaters.tooltip"
+                          )}
+                        >
+                        </ha-help-tooltip
+                      ></span>
+                      <span class="table"
+                        >${Object.entries(wrRepeaterRSSIMap).map(
+                          ([repeaterName, rssi]) => html`
+                            <div class="row">
+                              <span class="key-cell">${repeaterName}: </span>
+                              <span class="value-cell">${rssi}</span>
+                            </div>
+                          `
+                        )}</span
+                      >
+                    </div>`
+                  : ``}
+              </ha-expansion-panel>
+            `;
+          }
+          return ``;
+        })}
+      </ha-dialog>
+    `;
+  }
+
+  private _computeRSSI(rssi: number, includeUnit: boolean): string {
+    if (Object.values(RssiError).includes(rssi)) {
+      return this.hass.localize(
+        `ui.panel.config.zwave_js.rssi.rssi_error.${RssiError[rssi]}`
+      );
+    }
+    if (includeUnit) {
+      return `${rssi} ${this.hass.localize(
+        "ui.panel.config.zwave_js.rssi.unit"
+      )}`;
+    }
+    return rssi.toString();
+  }
+
+  private _computeDeviceNameById(device_id: string): string {
+    if (!this._devices) {
+      return device_id;
+    }
+    const device = this._devices.find((dev) => dev.id === device_id);
```

Comment:
If the statistics update a lot, it is better to create a map with `device_id` as key and the device (or just the name) as value instead of searching for the device every time.